setup.py

```diff
@@ -102,7 +102,7 @@ def run(self):
       data_files=data_files,
       cmdclass=cmdclass,
       # rosinstall dependency to be kept in order not to break ros hydro install instructions
-      install_requires=['vcstools>=0.1.37', 'pyyaml'],
+      install_requires=['vcstools>=0.1.37', 'pyyaml', 'pyaml'],
```

Comment:
Please look at the text at https://pypi.python.org/pypi/pyaml/15.8.2, which describes pyaml is not strictly necessary, it just make pretty pyYaml even prettier.